draft-ietf-tls-tls13.md

```diff
@@ -1938,11 +1938,14 @@ processed and transmitted as specified by the current active session state.
                case certificate:           Certificate;
                case certificate_verify:    CertificateVerify;
                case finished:              Finished;
-               case session_ticket:        NewSessionTicket;
+               case new_session_ticket:    NewSessionTicket;
                case key_update:            KeyUpdate;
            } body;
        } Handshake;
 
+[[TODO: Add IANA Considerations to rename the 'NewSessionTicket' HandshakeType
+value to 'new_session_ticket'.]]
+
```

Comment:
Sure. I just don't know how to write IANA considerations. But I'm happy to include the right sentence wherever.